src/nrniv/morphology/morphio_wrapper/morphio_wrapper.cpp

```diff
@@ -0,0 +1,472 @@
+#include "morphology/morphology.hpp"
+#include "morphio_wrapper.hpp"
+#include <morphio/morphology.h>
+#include <morphio/section.h>
+#include <morphio/soma.h>
+#include <fmt/core.h>
+
+namespace morphio {
+class Morphology;
+class Section;
+}  // namespace morphio
+
+
+namespace neuron::morphology {
+
+struct MorphIOWrapper::MorphIOWrapperImpl {
+    MorphIOWrapperImpl(morphio::Morphology&& morph)
+        : _morph{std::move(morph)}
+        , _sections{_morph.sections()} {}
+    ~MorphIOWrapperImpl() = default;
+    morphio::Morphology _morph;
+    std::vector<morphio::Section> _sections;
+};
+
+MorphIOWrapper::MorphIOWrapper(MorphIOWrapper&&) = default;
+MorphIOWrapper& MorphIOWrapper::operator=(MorphIOWrapper&&) = default;
+MorphIOWrapper::~MorphIOWrapper() = default;
+
+morphio::Point operator-(const morphio::Point& lhs, const morphio::Point& rhs) {
+    return morphio::Point{lhs[0] - rhs[0], lhs[1] - rhs[1], lhs[2] - rhs[2]};
+}
+
+morphio::floatType norm(morphio::Point p) {
+    return sqrt(p[0] * p[0] + p[1] * p[1]);
+}
+
+std::vector<morphio::floatType> norm(morphio::range<const morphio::Point> points) {
+    std::vector<morphio::floatType> norms(points.size());
+    std::transform(points.begin(), points.end(), norms.begin(), [](morphio::Point p) {
+        return norm(p);
+    });
+    return norms;
+}
+
+std::pair<std::vector<morphio::Point>, std::vector<morphio::floatType>>
+single_point_sphere_to_circular_contour(morphio::Morphology& neuron) {
+    /* Transform a single point soma that represents a sphere
+     * into a circular contour that represents the same sphere */
+    morphio::floatType radius = neuron.soma().diameters()[0] / 2.0;
+    constexpr int N = 20;
```

Comment:
Can we comment why 20, and maybe move that up in the scope, maybe the class declaration?